Release v11.3.0 (#932)
stellar/js-stellar-sdk: CHANGELOG.md

## Unreleased

## [v11.3.0](https://github.com/stellar/js-stellar-sdk/compare/v11.2.2...v11.3.0)

### Added
* Introduces an entire suite of helpers to assist with interacting with smart contracts ([#929](https://github.com/stellar/js-stellar-sdk/pull/929)):

>   - `ContractClient`: generate a class from the contract specification where each contract method gets included with a "JSified" name. Each method returns an `AssembledTransaction` that can be used to modify, simulate, decode results, and possibly sign, & submit the transaction.

Reviewer: The name is no longer JSified. Did you copy-paste this from one of my notes? I can send a PR to update the source.

```suggestion
  - `ContractClient`: generate a class from the contract specification, with a method for each original contract method. Each returns an `AssembledTransaction` that can be used to modify, simulate, decode results, and possibly sign, & submit the transaction.
```

Author: yeah I ripped this from the js docstrings

Author: fixed! thanks :+1:
